src/main/java/edu/byu/cs/autograder/git/GitHubAssignmentGrader.java

```diff
@@ -11,11 +11,20 @@
  * then this grader will give them full points.
  */
 public class GitHubAssignmentGrader {
+    private static final String RESUBMIT_PROMPT = " Push another commit to your repository and re-request grading to receive points for this assignment.";
+    private static final String FAILURE_MESSAGE = "Successfully fetched your repository, but the number of commits is insufficient." + RESUBMIT_PROMPT;
 
-    public Rubric.Results grade() throws DataAccessException {
+    public Rubric.Results grade(CommitVerificationResult commitVerificationResult) throws DataAccessException {
         RubricConfig rubricConfig = DaoService.getRubricConfigDao().getRubricConfig(Phase.GitHub);
         RubricConfig.RubricConfigItem configItem = rubricConfig.items().get(Rubric.RubricType.GITHUB_REPO);
-        return new Rubric.Results("Successfully fetched your repository", 1f, configItem.points(), null, null);
-    }
 
+        if(commitVerificationResult.verified()) {
+            return new Rubric.Results("Successfully fetched your repository and verified commits",
```

Comment:
It probably makes sense to extract this string into a `SUCCESS_MESSAGE` variable for consistency.